DateTimeCompareValidator.php

```diff
@@ -62,7 +62,11 @@ class DateTimeCompareValidator extends Validator
      * - `{compareAttribute}`: the label of the attribute to be compared with
      */
     public $message;
-
+    /**
+     * @var string Date format according to DateTime::createFromFormat specification.
+     *
```

Comment:
I think that "-" means that the extra newline has been deleted. On the editor i don't see an extra new line there.